tests/highlightui.js

```diff
@@ -205,6 +205,10 @@ describe( 'HighlightUI', () => {
 				return localizedEditor();
 			} );
 
+			afterEach( () => {
+
+			} );
+
```

Comment:
Do we need that empty block?

Reply:
Not really, it's a mistake - will fix it in a minute.